e2e-tests/custom-replset-name/conf/some-name.yml

```diff
@@ -0,0 +1,230 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: some-name
+spec:
+  crVersion: 1.14.0
+  allowUnsafeConfigurations: true
+  backup:
+    enabled: true
+    image: percona/percona-backup-mongodb:2.0.4
+    pitr:
+      enabled: false
+    serviceAccountName: percona-server-mongodb-operator
+    storages:
+      aws-s3:
+        type: s3
+        s3:
+          credentialsSecret: aws-s3-secret
+          region: us-east-1
+          bucket: operator-testing
+          prefix: psmdb
+      minio:
+        type: s3
+        s3:
+          credentialsSecret: minio-secret
+          region: us-east-1
+          bucket: operator-testing
+          endpointUrl: http://minio-service:9000/
+      gcp-cs:
+        type: s3
+        s3:
+          credentialsSecret: gcp-cs-secret
+          region: us-east-1
+          bucket: operator-testing
+          prefix: psmdb
+          endpointUrl: https://storage.googleapis.com
+  image: percona/percona-server-mongodb:4.4.10-11
+  imagePullPolicy: Always
+  mongod:
```

Comment:
@egegunes, we do not have mongod at all now. That is why you have a problem with this test.